### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by adamclasic**
In this branch, we have setup a project as follows :
- added lib folder to hold the game logic
- added bin folder to hold the executable file called main.rb.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env ruby
```
Comment (majovanilla) [inline bin/main.rb]: Your main.rb file should puts "Hello world" ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from Pathwright)

It should be executable on the terminal.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey team , 👋

You missed the purpose of the milestone!, besides creating the folder structure. Please check the commented code below and request a new review. :smiley: 

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: bin/main.rb (hunk 1 above)
Comment (PhillipUg) [reply bin/main.rb]: Oh, sori. we must have missed that. Thanks for pointing it out

**Comment by PhillipUg** [comment]
What we have fixed: 
- made main.rb executable
- made main.rb output "Hello World"

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the milestone is approved :heavy_check_mark: You may merge :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Pull request merged**